indico/web/client/js/jquery/widgets/jinja/permissions_widget.js

```diff
@@ -159,11 +161,15 @@ import Palette from 'indico/utils/palette';
       return $buttonsGroup;
     },
     _renderEditBtn(principal, permissions) {
-      if (principal._type === 'IPNetworkGroup') {
+      if (principal._type === 'IPNetworkGroup' || principal._type === 'RegistrationForm') {
+        const $title = {
```

Comment:
just `title` since it's not a jquery object (which is why some other vars use `$something`)